ui/packages/gallery/src/Gallery.svelte

```diff
@@ -293,8 +293,34 @@
 
 	.grid-container {
 		display: grid;
+		grid-template-columns: var(--grid-cols);
 		gap: var(--size-2);
 	}
+	@media (min-width: 640px) {
+		.grid-container {
+			grid-template-columns: var(--sm-grid-cols);
+		}
+	}
+	@media (min-width: 768px) {
+		.grid-container {
+			grid-template-columns: var(--md-grid-cols);
+		}
+	}
+	@media (min-width: 1024px) {
+		.grid-container {
+			grid-template-columns: var(--lg-grid-cols);
+		}
+	}
+	@media (min-width: 1280px) {
+		.grid-container {
+			grid-template-columns: var(--xl-grid-cols);
+		}
+	}
+	@media (min-width: 1536px) {
+		.grid-container {
+			grid-template-columns: var(--2xl-grid-cols);
+		}
+	}
```

Comment:
Use the media query variables here instead of hardcoded values:

```
--screen-sm: (min-width: 640px)
--screen-md: (min-width: 768px)
--screen-lg: (min-width: 1024px)
--screen-xl: (min-width: 1280px)
--screen-xxl: (min-width: 1536px)
```

You can just do this:

```css
@media (--screen-md) {
 ...
}
```

etc.

Reply:
Oh neat, done.